packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -27,43 +27,34 @@ type ALUIEvent<T = EventHandlerMap> = {
     event: K,
     // Element target associated with the domEvent
     element: HTMLElement | null,
-    // Element text extracted from element
-    elementName?: string | null,
-    // The source attribute where we got the elementName from
-    elementNameSource?: ALElementNameResult['source'] | null,
     // Whether the event is generated from a user action or dispatched via script
     isTrusted: boolean,
+
+    autoLoggingID: ALID | null,
   }>
 }[keyof T];
 
 export type ALUIEventCaptureData = Readonly<
   ALUIEvent &
   ALFlowletEvent &
   ALReactElementEvent &
+  ALElementTextEvent &
   {
     captureTimestamp: number,
     surface: string | null,
-    autoLoggingID: ALID | null,
   }
 >;
 
 export type ALUIEventBubbleData = Readonly<
   ALUIEvent &
   {
     bubbleTimestamp: number,
-    autoLoggingID: ALID | null,
   }
 >;
 
 export type ALLoggableUIEvent = Readonly<
-  ALTimedEvent &
-  ALUIEvent &
-  ALFlowletEvent &
-  ALReactElementEvent &
-  {
-    autoLoggingID: ALID | null,
-    surface?: string | null,
-  }
+  Omit<ALUIEventCaptureData, 'captureTimestamp'> &
+  ALTimedEvent
 >;
 
 export type ALUIEventData = Readonly<
```

Comment:
In this file we use `getInteractable` with `true` in the capture phase but not in the bubble phase,  maybe that can cause some weirdness in the output

Reply:
I will send an update soon to make sure the logic is shared between capture and buble, and test to see if data is correct. Thanks for noticing the difference.